deployments/container/vulkan/Dockerfile

```diff
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
whats this?

Reply:
It's a tag to improve machine-readability of the license headers.

See https://spdx.dev/learn/handling-license-info/